posthog/api/instance_status.py

```diff
@@ -44,6 +46,17 @@ def list(self, request: Request) -> Response:
 
         metrics.append({"key": "posthog_git_sha", "metric": "PostHog Git SHA", "value": GIT_SHA})
 
+        helm_info = json.loads(os.getenv("HELM_INSTALL_INFO", "{}"))
+        if len(helm_info) > 0:
+            metrics.append(
```

Comment:
Note: If you wanted to expose it at top-level (which is ok) you could always do 

```
metrics.append(
            {"key": "chart_version", "metric": "Chart version", "value": get_helm_info_env().get("chart_version", "Undefined"),}
```

Reply:
I think it's fine, I considered that, but this way the neat thing about this is if we add anything to HELM_INSTALL_INFO it will show up